docs/README.dev.md

```diff
@@ -80,52 +85,28 @@ bumpversion patch
 
 ## Making a release
 
-This section describes how to make a release in 3 parts:
-
-1. preparation
-1. making a release on PyPI
-1. making a release on GitHub
+This section describes how to make a release in 3 parts: preparation, release and validation.
 
-### (1/3) Preparation
+### Preparation
 
-1. Update the <CHANGELOG.md> (don't forget to update links at bottom of page)
-2. Verify that the information in `CITATION.cff` is correct, and that `.zenodo.json` contains equivalent data
+1. Update the `CHANGELOG.md` file
+2. Verify that the information in `CITATION.cff` is correct
 3. Make sure the [version has been updated](#versioning).
 4. Run the unit tests with `hatch run test`
 
-### (2/3) PyPI
+### Making the GitHub release
 
-First prepare a new directory, for example:
-```shell
-cd $(mktemp -d lilio.XXXXXX)
-```
+Make a release and tag on GitHub.com. This will:
 
-A fresh git clone ensures the release has the state of origin/main branch
+ - trigger Zenodo into making a snapshot of your repository and sticking a DOI on it.
+ - start a GitHub action that builds and uploads the new version to [PyPI](https://pypi.org/project/lilio/).
+    - Which should trigger [conda-forge](https://anaconda.org/conda-forge/lilio) to update the package as well.
 
-```shell
-git clone https://github.com/AI4S2S/lilio .
-```
 
-In a your terminal, with an activated environment which has [`hatch`](https://hatch.pypa.io/latest/) installed do:
-
-```shell
-pip install hatch --upgrade
-hatch build
-```
-
-If the build was succesfull, publish it to [PyPI's test servers](https://test.pypi.org/). Note that your credentials are different between test.pypi.org and pypy.org.
-```shell
-hatch publish --repo test
-```
-
-Visit [https://test.pypi.org/project/lilio](https://test.pypi.org/project/lilio) and
-verify that your package was uploaded successfully.
-
-Now we can publish to PyPI:
-```
-hatch publish
-```
+### Validation
 
-### (3/3) GitHub
+After making the release, you should check that:
 
-Don't forget to also make a [release on GitHub](https://github.com/AI4S2S/lilio/releases/new). If your repository uses the GitHub-Zenodo integration this will also trigger Zenodo into making a snapshot of your repository and sticking a DOI on it.
+1. The [Zenodo page](https://doi.org/10.5281/zenodo.7620212) is updated
```

Comment:
Not sure if it is necessary to provide a link here or not as each time when a new snapshot is generated on zenodo, we will have a new doi and therefore a new link to it.

```suggestion
1. The Zenodo page is updated
```

Reply:
This is the persistent DOI, it will forward to the latest released version.

Reply:
Oh yes! Just notice the small notice on the page about "cite all versions". Cool!